task-sdk/src/airflow/sdk/execution_time/comms.py

```diff
@@ -304,7 +304,7 @@ class AssetEventSourceTaskInstance:
     def xcom_pull(
         self,
         *,
-        key: str = "return_value",  # TODO: Make this a constant; see RuntimeTaskInstance.
+        key: str = "return_value",
```

Comment:
We've got the constant now don't we?
```suggestion
        key: str = BaseXCom.XCOM_RETURN_KEY,
```

Reply:
We do yes, but this is in comms so it was running into a circular import because xcom module imports comms and vice versa. I tried moving the imports inside functions but some of those have it in return type hints :)

Reply:
Resolving for now, let me know if you disagree. Can figure out another way